### PR TITLE
Add '<browse>' links matching those in AdminUI

### PR DESCRIPTION
**Pull request by aydun**
Overview
----------------------------------------
This adds '<browse>' paths to the schema xml for use by https://github.com/civicrm/civicrm-core/pull/26887 for paths used in AdminUI.

Before
----------------------------------------
Not redirected to a browse page in some cases.

After
----------------------------------------
Redirected in more situations. 

Technical Details
----------------------------------------

Comments
----------------------------------------
@colemanw  Some of these work (ie following the sequence in https://github.com/civicrm/civicrm-core/pull/26887 goes to the browse page), but some don't.  Eg: contact subtype, relationship type, financial accounts

### CODE REVIEW

**Comment by aydun** [comment]
Thanks @colemanw - that worked

File: xml/schema/Core/LocationType.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <add>civicrm/admin/locationType/edit?action=add&amp;reset=1</add>
     <update>civicrm/admin/locationType/edit?action=update&amp;id=[id]&amp;reset=1</update>
     <delete>civicrm/admin/locationType/edit?action=delete&amp;id=[id]&amp;reset=1</delete>
+    <browse>civicrm/admin/locationType</browse>
```
Comment (colemanw) [inline xml/schema/Core/LocationType.xml:14]: @aydun it's already here!

Comment (colemanw) [reply xml/schema/Core/LocationType.xml:14]: But good to see the script gives the exact same output as the hand-added one :)

File: CRM/Core/DAO/LocationType.php [hunk 2]
```diff
@@ -43,7 +43,10 @@ class CRM_Core_DAO_LocationType extends CRM_Core_DAO {
    * @var string[]
    */
   protected static $_paths = [
-    'browse' => 'civicrm/admin/locationType',
+    'browse' => [
+      'civicrm/admin/locationType',
+      'civicrm/admin/locationType',
+    ],
```
Comment (colemanw) [inline CRM/Core/DAO/LocationType.php:49]: That looks funny 😆

File: xml/schema/Core/LocationType.xml (hunk 1 above)
Comment (aydun) [reply xml/schema/Core/LocationType.xml:13]: Grr!  The script was supposed to recognise that...  it does now.

**Pull request merged**